evidently/analyzers/data_drift_analyzer.py

```diff
@@ -22,28 +24,94 @@ def dataset_drift_evaluation(p_values, drift_share=0.5):
 PValueWithConfidence = collections.namedtuple("PValueWithConfidence", ["p_value", "confidence"])
 
 
+@dataclass
+class DataDriftAnalyzerFeatureMetrics:
+    current_small_hist: list
+    ref_small_hist: list
+    feature_type: str
+    p_value: float
+
+    def as_dict(self) -> Dict[str, Any]:
```

Comment:
As far as I can see `as_dict` is only used in `profile_section` (as intended, as it should return JSON as result).

Probably better to move these methods inside `profile_section` to place where it used (and purpose of this methods).

Reply:
done - to-dict serialisation logic was moved to `profile_section` to DataDriftProfileSection

also I moved the test for dict result structure and leave separate test for DataDriftAnalyzer